wormhole-connect/src/sdk/gasEstimates.ts

```diff
@@ -69,6 +69,10 @@ const estimateGasFee = async (
     return toFixedDecimals(utils.formatUnits(gasEstimates.sendToken, 9), 6);
   }
 
+  if (fromChainId === MAINNET_CHAINS.sei) {
+    return toFixedDecimals(utils.formatUnits(gasEstimates.sendToken, 6), 6);
+  }
```

Comment:
Didn't we add a `nativeTokenDecimals` on the network config? I think that's what's happening here anyway...

Reply:
I haven't found any reference to `nativeTokenDecimals` on the repo. Do you mean the `decimals` field?

Reply:
Hm, it looks like it hasn't been added.  I think it should be added

Reply:
I've left out the decimals rework for a separate PR. I think we can include the `nativeTokenDecimals` to this one as well.